### PR TITLE
add docs for user-manual for volto frontend

### PR DESCRIPTION
**Pull request by MAX-786**
#3827 
This is the proposed structure or outline of first page for user-manual docs for volto frontend. It is only a start to have something to look at and discuss. Feedback welcome.

Proposed:
[Screencast from 03-12-22 03:57:46 PM IST.webm](https://user-images.githubusercontent.com/99530996/205436446-22477725-7ecf-4155-8855-35b11686e40b.webm)

I, Mohammad Hussain, agree to have this contribution published under Creative Commons 4.0 International License (CC BY 4.0), with attribution to the Plone Foundation.

### CODE REVIEW

File: docs/source/user-manual/blocks.md
Comment (stevepiercy) [inline docs/source/user-manual/blocks.md:7]:
```suggestion
    "keywords": "Volto, Plone, frontend, React, User manual, edit blocks"
```

File: docs/source/user-manual/index.md
Comment (stevepiercy) [inline docs/source/user-manual/index.md:12]: The second sentence is not yet true. When it is true, then we can add it.

```suggestion
This is the user manual for Volto, the Plone 6 frontend.
```

File: docs/source/user-manual/index.md
Comment (stevepiercy) [inline docs/source/user-manual/index.md:10]: It is helpful to use labels, which allow authors to reference the label from arbitrary locations in the rest of the docs with the [`ref` directive](https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html#targets-and-cross-referencing).

```suggestion

(user-manual-label)=

# User Manual
```

File: docs/source/user-manual/index.md
Comment (stevepiercy) [inline docs/source/user-manual/index.md:13]: Let's add a todo for future PRs. Once it is merged, it will help guide authors.

````suggestion
```{todo}
The Plone 5.2 documentation for Classic UI is well structured, and should be emulated as the model for the Volto User Manual.

- [Working with Content](https://docs.plone.org/working-with-content/index.html)

Additional sources of content may be used to fill out the Volto User Manual.
Note that the audience for these sources may be a Developer, not an Editor, and therefore might need to be retargeted.

- [Volto Hands-On](https://training.plone.org/voltohandson/index.html)
- [Plone 6 Demo](https://6.demo.plone.org/) for screen shots and videos.

```
````

File: docs/source/user-manual/index.md
Comment (stevepiercy) [inline docs/source/user-manual/index.md:4]:
```suggestion
    "description": "Volto User Manual for Plone 6"
```

File: docs/source/user-manual/index.md
Comment (stevepiercy) [inline docs/source/user-manual/index.md:5]:
```suggestion
    "property=og:description": "Volto User Manual for Plone 6"
```

File: docs/source/user-manual/blocks.md
Comment (stevepiercy) [inline docs/source/user-manual/blocks.md:17]:
```suggestion
## Choose a pre-built block type
```

File: docs/source/user-manual/blocks.md
Comment (stevepiercy) [inline docs/source/user-manual/blocks.md:19]:
````suggestion
```{todo}
Add description of a Volto block.
```

Volto offers several default block types out of the box.
You can access and choose a block type to add to your content type when you have an empty block in it.

To create an empty block after an existing block, click in the block, then hit the {kbd}`Enter` key.
A new empty block appears.

```{todo}
Add screenshots or video illustrating the steps.
```

```{note}
There is a new experimental feature that places a `+` below every block in a content type.

See https://github.com/plone/volto/pull/3815 for details of the feature and how to enable it.
```

Now with your empty block available, you can select its type in one of two ways.
````

File: docs/source/user-manual/blocks.md
Comment (stevepiercy) [inline docs/source/user-manual/blocks.md:20]:
```suggestion
1.  Click the `+` button on the left-hand side of the empty block.
```

File: docs/source/user-manual/blocks.md
Comment (stevepiercy) [inline docs/source/user-manual/blocks.md:21]:
```suggestion
2. Type `/` inside empty block to open the blocks menu.
    ![Screen Shot 2022-12-03 at 2 52 11 PM](https://user-images.githubusercontent.com/102112/205465439-842aac3a-5fbc-4bb3-a4db-6ff687068467.png)
```

Use this screenshot, nested within the list item.

Screen Shot 2022-12-03 at 2 52 11 PM

File: docs/source/user-manual/blocks.md
Comment (stevepiercy) [reply docs/source/user-manual/blocks.md:20]: Use this screenshot, nested within the list item.

Screen Shot 2022-12-03 at 2 50 47 PM

File: docs/source/user-manual/blocks.md
Comment (stevepiercy) [inline docs/source/user-manual/blocks.md:26]:
```suggestion
A description block provides a blank textarea to write plain text.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
I think this is a good start. Would you please take a look at my suggestions, and either accept them or respond to them, as needed?

I also thought some more on the future direction of the Volto User Manual, and added my thoughts accordingly. This is just a start, but gives us something to consider at this moment.

Thank you!

**Comment by MAX-786** [comment]
Thanks, @stevepiercy 😊 for your thoughtful input, I'll look into these changes and will respond accordingly.

File: docs/source/user-manual/blocks.md
Comment (MAX-786) [reply docs/source/user-manual/blocks.md:5]: Ohh thanks this was something new to me.

File: docs/source/user-manual/blocks.md
Comment (MAX-786) [reply docs/source/user-manual/blocks.md:15]: ohh I heard about 79 column rule but "Docs are not code" mantra is new to me 😅😂

File: docs/source/user-manual/index.md
Comment (MAX-786) [reply docs/source/user-manual/index.md:10]: sorry I may have forgotten to add it here.

**Review by MAX-786 — COMMENTED** [review]
Thanks, Steve, for reviewing this PR and correcting it. I have also added screenshots to the `/_static` folder, but in the future we will be adding more illustrating screenshots, so I think it will be better to move all these into a separate folder named `user-manual` in another PR. There were many things I did not know before and many silly mistakes I made, but I'll make sure not to repeat them next time🙂.

**Comment by stevepiercy** [comment]
@MAX-786 images should be placed within `_static`, and within a subdirectory as appropriate. As such, let's do it the "right" way from the start, and move them into `_static/user-manual`, and even deeper directories as needed. We will need to ensure that they get copied over in the main `plone/documentation` repo, too. It is tricky to get the paths just right.

**Comment by MAX-786** [comment]
Hello @stevepiercy , I have already signed the contributing agreement and mailed the scanned copy to agreement@plone.org with my Name as subject, please let me know if there are any other steps I should be following.

**Comment by MAX-786** [comment]
@stevepiercy oh sorry 😅, i think my comment was misleading, I was also asking the same that should we move all images related to user manual inside `_static/user-manual`, I'll do it right now.
Also if I am getting it right, for updating `plone/documentation` do we have to use command `git submodule update --remote` after this PR is merged and make a seperate PR for that?

**Comment by polyester** [comment]
Contributor agreement for @MAX-786 has been received and processed

**Comment by MAX-786** [comment]
I think there's some mistake from my side in configuration of my user email in my local machine while I made this PR which is fixed now but bot is not able to recognize it.That's why I am closing this PR and opening the new one with same comment.Sorry for the inconvenience.

**Comment by stevepiercy** [comment]
> Also if I am getting it right, for updating `plone/documentation` do we have to use command `git submodule update --remote` after this PR is merged and make a seperate PR for that?

You do not. Maintainers do.

**Pull request closed without merging**
